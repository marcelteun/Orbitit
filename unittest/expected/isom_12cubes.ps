%!PS-Adobe-2.0
%%Title: SimpleShape
%%Creator: PS.py written by marcelteun
%%Pages: 1 1
%%BoundingBox: 0 0 595 842
%%EndComments
%%BeginSetup
%%EndSetup
%%Page: 1 1
%%BeginPage
10.000000 10.000000 translate
/vertices [
  [-1 1] % 0
  [-1 -1] % 1
  [1 -1] % 2
  [1 1] % 3
  [0.8476042 -0.5847604] % 4
  [1 -0.1411934] % 5
  [1 0.2780641] % 6
  [-0.7655876 0.875299] % 7
  [-1 0.0809817] % 8
  [0.8476042 -0.5847604] % 9
  [-0.7655876 0.875299] % 10
  [-1 0.1930124] % 11
  [0.7206545 0.7652357] % 12
  [-1 -0.2114686] % 13
  [-1 -0.4258093] % 14
  [0.7071436 -0.7525513] % 15
  [0.7071436 -0.7525513] % 16
  [0.7206545 0.7652357] % 17
  [-0.8133541 0.6190106] % 18
  [0.3239268 -1] % 19
  [0.5935257 -1] % 20
  [0.5755077 0.8637769] % 21
  [0.5755077 0.8637769] % 22
  [-0.8133541 0.6190106] % 23
  [1 0.1411934] % 24
  [0.2474487 1] % 25
  [-0.2752551 1] % 26
  [-1 0.3484692] % 27
  [-1 0.3484692] % 28
  [1 -0.5505103] % 29
  [-0.2474487 1] % 30
  [-0.6515308 -0.797959] % 31
  [-0.6515308 -0.797959] % 32
  [0.4870453 -0.8847161] % 33
  [0.4870453 -0.8847161] % 34
  [0.2752551 1] % 35
  [1 0] % 36
  [0 1] % 37
  [-1 -1] % 38
  [1 0] % 39
  [0 1] % 40
  [-1 -1] % 41
  [-0.7694321 0.6541482] % 42
  [-1 0.2457187] % 43
  [-1 -0.2576539] % 44
  [0.3239268 -1] % 45
  [0.948181 -0.3717613] % 46
  [-0.7694321 0.6541482] % 47
  [0.5935257 -1] % 48
  [0.948181 -0.3717613] % 49
  [0 -1] % 50
  [1 -0.1835034] % 51
  [1 -0.1835034] % 52
  [-0.4494897 1] % 53
  [-0.8164966 1] % 54
  [0 -1] % 55
  [-0.2787754 -1] % 56
  [0.8989795 0.5191836] % 57
  [0.8989795 0.5191836] % 58
  [-0.6704941 0.7445489] % 59
  [-0.6704941 0.7445489] % 60
  [-0.4 -1] % 61
  [-0.2787754 -1] % 62
  [1 -0.0625725] % 63
  [-1 -0.0696938] % 64
  [-0.4 -1] % 65
  [1 0.5505103] % 66
  [-1 -0.0696938] % 67
  [-1 -0.1835034] % 68
  [0 -1] % 69
  [0.4494897 1] % 70
  [-1 -0.1835034] % 71
  [0 -1] % 72
  [0.8164966 1] % 73
] def
/faces [
  [0 1 2 3][4 5][6 7][8 9][10 11][12 13][14 15][16 17][18 19][20 21][22 23]
  [24 25][26 27][28 29][30 31][32 33][34 35][36 37][38 39][40 41][42 43][44 45]
  [46 47][48 49][50 51][52 53][54 55][56 57][58 59][60 61][62 63][64 65][66 67]
  [68 69][70 71][72 73]
] def
/bbox [-1 -1 1 1] def
/scalingSize 50 def
.1 setlinewidth

/dx bbox 2 get bbox 0 get sub scalingSize mul def
/dy bbox 3 get bbox 1 get sub scalingSize mul def

gsave
  bbox 0 get scalingSize mul neg
  bbox 1 get scalingSize mul neg
  translate
  faces aload length { %repeat
    dup 0 get
    vertices exch get
    aload pop
    scalingSize mul exch
    scalingSize mul exch
    moveto
    aload length 1 sub { %repeat
      vertices exch get
      aload pop
      scalingSize mul exch
      scalingSize mul exch
      lineto
    } repeat
    pop
    closepath stroke
  } repeat


grestore
105.000000 0 translate
/vertices [
  [1 1] % 0
  [-1 1] % 1
  [1 -1] % 2
  [-1 -1] % 3
  [0.7206545 0.7652357] % 4
  [-1 -0.2114686] % 5
  [-1 -0.4258093] % 6
  [0.7071436 -0.7525513] % 7
  [0.7071436 -0.7525513] % 8
  [0.7206545 0.7652357] % 9
  [1 0.2780641] % 10
  [-0.7655876 0.875299] % 11
  [0.8476042 -0.5847604] % 12
  [1 -0.1411934] % 13
  [-0.7655876 0.875299] % 14
  [-1 0.1930124] % 15
  [-1 0.0809817] % 16
  [0.8476042 -0.5847604] % 17
  [-0.7694321 0.6541482] % 18
  [-1 0.2457187] % 19
  [0.948181 -0.3717613] % 20
  [-0.7694321 0.6541482] % 21
  [0.5935257 -1] % 22
  [0.948181 -0.3717613] % 23
  [-1 -0.2576539] % 24
  [0.3239268 -1] % 25
  [-0.2474487 1] % 26
  [-0.6515308 -0.797959] % 27
  [-0.6515308 -0.797959] % 28
  [0.4870453 -0.8847161] % 29
  [0.4870453 -0.8847161] % 30
  [0.2752551 1] % 31
  [1 0.1411934] % 32
  [0.2474487 1] % 33
  [-0.2752551 1] % 34
  [-1 0.3484692] % 35
  [-1 0.3484692] % 36
  [1 -0.5505103] % 37
  [-1 -1] % 38
  [1 0] % 39
  [0 1] % 40
  [-1 -1] % 41
  [1 0] % 42
  [0 1] % 43
  [0.5935257 -1] % 44
  [0.5755077 0.8637769] % 45
  [-0.8133541 0.6190106] % 46
  [0.3239268 -1] % 47
  [0.5755077 0.8637769] % 48
  [-0.8133541 0.6190106] % 49
  [-1 -0.1835034] % 50
  [0 -1] % 51
  [0 -1] % 52
  [0.8164966 1] % 53
  [0.4494897 1] % 54
  [-1 -0.1835034] % 55
  [-1 -0.0696938] % 56
  [-0.4 -1] % 57
  [1 0.5505103] % 58
  [-1 -0.0696938] % 59
  [-0.2787754 -1] % 60
  [1 -0.0625725] % 61
  [-0.2787754 -1] % 62
  [0.8989795 0.5191836] % 63
  [-0.6704941 0.7445489] % 64
  [-0.4 -1] % 65
  [0.8989795 0.5191836] % 66
  [-0.6704941 0.7445489] % 67
  [0 -1] % 68
  [1 -0.1835034] % 69
  [1 -0.1835034] % 70
  [-0.4494897 1] % 71
  [-0.8164966 1] % 72
  [0 -1] % 73
] def
/faces [
  [0 1 3 2][4 5][6 7][8 9][10 11][12 13][14 15][16 17][18 19][20 21][22 23]
  [24 25][26 27][28 29][30 31][32 33][34 35][36 37][38 39][40 41][42 43][44 45]
  [46 47][48 49][50 51][52 53][54 55][56 57][58 59][60 61][62 63][64 65][66 67]
  [68 69][70 71][72 73]
] def
/bbox [-1 -1 1 1] def
/scalingSize 50 def
.1 setlinewidth

/dx bbox 2 get bbox 0 get sub scalingSize mul def
/dy bbox 3 get bbox 1 get sub scalingSize mul def

gsave
  bbox 0 get scalingSize mul neg
  bbox 1 get scalingSize mul neg
  translate
  faces aload length { %repeat
    dup 0 get
    vertices exch get
    aload pop
    scalingSize mul exch
    scalingSize mul exch
    moveto
    aload length 1 sub { %repeat
      vertices exch get
      aload pop
      scalingSize mul exch
      scalingSize mul exch
      lineto
    } repeat
    pop
    closepath stroke
  } repeat


grestore
105.000000 0 translate
/vertices [
  [-0.5797959 1.2898979] % 0
  [1.2898979 0.5797959] % 1
  [-1.2898979 -0.5797959] % 2
  [0.5797959 -1.2898979] % 3
  [1.0860309 0.0430155] % 4
  [0.2793455 0.9635996] % 5
  [-0.0951036 -1.0335739] % 6
  [1.0099291 -0.1573603] % 7
  [0.174614 1.0033762] % 8
  [-0.4870453 -0.8847161] % 9
  [0.5527418 0.8597648] % 10
  [-0.83612 0.6149984] % 11
  [-0.9849778 0.2230568] % 12
  [0.8663177 -0.535488] % 13
  [0.9060943 -0.4307565] % 14
  [0.7531175 0.7836629] % 15
  [1.0263272 -0.1141841] % 16
  [-0.2593526 1.0554856] % 17
  [-0.2593526 1.0554856] % 18
  [-1 0.0809817] % 19
  [-1 0.0809817] % 20
  [0.8476042 -0.5847604] % 21
  [0.0292856 1.0585713] % 22
  [-0.8696938 -0.7393877] % 23
  [0.8696938 0.5651531] % 24
  [0.0292856 1.0585713] % 25
  [-0.2230568 -0.9849778] % 26
  [0.8696938 0.5651531] % 27
  [1 0.2752551] % 28
  [-0.8847161 0.4870453] % 29
  [0.8111226 -0.6808164] % 30
  [1 0.2752551] % 31
  [-1.1303062 -0.1595918] % 32
  [0.8111226 -0.6808164] % 33
  [-0.355051 -0.9348469] % 34
  [1.2898979 0.5797959] % 35
  [-0.9348469 0.355051] % 36
  [-0.355051 -0.9348469] % 37
  [1.2898979 0.5797959] % 38
  [-0.9348469 0.355051] % 39
  [0.1253416 1.0220896] % 40
  [-0.6032723 -0.8016362] % 41
  [-0.6032723 -0.8016362] % 42
  [0.6172062 -0.894634] % 43
  [0.6172062 -0.894634] % 44
  [0.5959179 0.8433667] % 45
  [-0.5265986 -0.8696938] % 46
  [1 -0.1835034] % 47
  [-0.4494897 1] % 48
  [-0.5265986 -0.8696938] % 49
  [1 -0.1835034] % 50
  [-0.4494897 1] % 51
  [0.1680816 -1.0638367] % 52
  [0.9595918 -0.2898979] % 53
  [0.9595918 -0.2898979] % 54
  [-0.7393877 0.8696938] % 55
  [-0.9570634 0.2965553] % 56
  [0.1680816 -1.0638367] % 57
  [-0.6840408 0.8319184] % 58
  [-0.4135467 -0.9126305] % 59
  [0.1595918 -1.1303062] % 60
  [0.4202041 0.9101021] % 61
  [0.4202041 0.9101021] % 62
  [-0.6840408 0.8319184] % 63
  [-1 0.1835034] % 64
  [0.4494897 -1] % 65
  [0.4494897 -1] % 66
  [0.5265986 0.8696938] % 67
  [0.5265986 0.8696938] % 68
  [-1 0.1835034] % 69
] def
/faces [
  [1 0 2 3][4 5][6 7][8 9][10 11][12 13][14 15][16 17][18 19][20 21][22 23]
  [24 25][26 27][28 29][30 31][32 33][34 35][36 37][38 39][40 41][42 43][44 45]
  [46 47][48 49][50 51][52 53][54 55][56 57][58 59][60 61][62 63][64 65][66 67]
  [68 69]
] def
/bbox [-1.2898979 -1.2898979 1.2898979 1.2898979] def
/scalingSize 50 def
.1 setlinewidth

/dx bbox 2 get bbox 0 get sub scalingSize mul def
/dy bbox 3 get bbox 1 get sub scalingSize mul def

gsave
  bbox 0 get scalingSize mul neg
  bbox 1 get scalingSize mul neg
  translate
  faces aload length { %repeat
    dup 0 get
    vertices exch get
    aload pop
    scalingSize mul exch
    scalingSize mul exch
    moveto
    aload length 1 sub { %repeat
      vertices exch get
      aload pop
      scalingSize mul exch
      scalingSize mul exch
      lineto
    } repeat
    pop
    closepath stroke
  } repeat


grestore
133.989795 0 translate
/vertices [
  [-0.4759592 1.3317143] % 0
  [1.3317143 0.4759592] % 1
  [-1.3317143 -0.4759592] % 2
  [0.4759592 -1.3317143] % 3
  [0.3239268 -1] % 4
  [0.9611421 0.3776126] % 5
  [0.9611421 0.3776126] % 6
  [-0.7216425 0.8127396] % 7
  [-0.8133541 0.6190106] % 8
  [0.3239268 -1] % 9
  [-0.9864224 0.2534259] % 10
  [-1.066487 -0.4635497] % 11
  [1.0163017 0.1658571] % 12
  [-0.938487 0.3546833] % 13
  [0.9642502 -0.3002617] % 14
  [1.0163017 0.1658571] % 15
  [-1.066487 -0.4635497] % 16
  [0.7848593 -0.6792021] % 17
  [0.7206545 0.7652357] % 18
  [-0.7935924 0.6607546] % 19
  [1.0160692 -0.0696938] % 20
  [0.9643279 0.6498804] % 21
  [-1.0089742 0.2057879] % 22
  [-0.9753363 -0.2620204] % 23
  [-0.9753363 -0.2620204] % 24
  [1.0160692 -0.0696938] % 25
  [0.8187595 0.5912431] % 26
  [-0.2474487 1] % 27
  [-0.1790918 -1.0216122] % 28
  [0.8187595 0.5912431] % 29
  [-0.2474487 1] % 30
  [-0.6515308 -0.797959] % 31
  [1.1393877 0.0696938] % 32
  [-1.0529389 0.1129183] % 33
  [-1.0529389 0.1129183] % 34
  [-0.6766632 -0.7860612] % 35
  [-0.2042243 -1.0097145] % 36
  [0.8434232 -0.5554933] % 37
  [-0.4278775 -0.9038367] % 38
  [0.9038367 -0.4278775] % 39
  [-0.4759592 1.3317143] % 40
  [-0.4278775 -0.9038367] % 41
  [0.9038367 -0.4278775] % 42
  [-0.4759592 1.3317143] % 43
  [-1 -0.2114686] % 44
  [0.1505742 -1.0269601] % 45
  [0.7206545 0.7652357] % 46
  [-1 -0.2114686] % 47
  [0.1505742 -1.0269601] % 48
  [0.9643279 0.6498804] % 49
  [0.3101021 -1.2531973] % 50
  [0.4278775 0.9038367] % 51
  [-0.8253197 0.5937347] % 52
  [-0.0216122 -1.0961633] % 53
  [0.4278775 0.9038367] % 54
  [-0.8253197 0.5937347] % 55
  [0.9306101 -0.3713222] % 56
  [0.1759101 1.0231184] % 57
  [-0.8740163 0.4908694] % 58
  [0.6682857 -0.9254489] % 59
  [0.0663429 1.0749877] % 60
  [-0.8740163 0.4908694] % 61
  [-0.9245929 -0.3860612] % 62
  [0.5903837 -0.8539103] % 63
  [0.5903837 -0.8539103] % 64
  [0.1759101 1.0231184] % 65
  [0.0663429 1.0749877] % 66
  [-0.9245929 -0.3860612] % 67
  [0.4278775 0.9038367] % 68
  [-1.1658571 -0.5544762] % 69
  [-0.8341429 -0.7115102] % 70
  [0.9823537 -0.2620204] % 71
  [0.9823537 -0.2620204] % 72
  [0.4278775 0.9038367] % 73
] def
/faces [
  [1 0 2 3][4 5][6 7][8 9][10 11][12 13][14 15][16 17][18 19][20 21][22 23]
  [24 25][26 27][28 29][30 31][32 33][34 35][36 37][38 39][40 41][42 43][44 45]
  [46 47][48 49][50 51][52 53][54 55][56 57][58 59][60 61][62 63][64 65][66 67]
  [68 69][70 71][72 73]
] def
/bbox [-1.3317143 -1.3317143 1.3317143 1.3317143] def
/scalingSize 50 def
.1 setlinewidth

/dx bbox 2 get bbox 0 get sub scalingSize mul def
/dy bbox 3 get bbox 1 get sub scalingSize mul def

gsave
  bbox 0 get scalingSize mul neg
  bbox 1 get scalingSize mul neg
  translate
  faces aload length { %repeat
    dup 0 get
    vertices exch get
    aload pop
    scalingSize mul exch
    scalingSize mul exch
    moveto
    aload length 1 sub { %repeat
      vertices exch get
      aload pop
      scalingSize mul exch
      scalingSize mul exch
      lineto
    } repeat
    pop
    closepath stroke
  } repeat


grestore
138.171426 0 translate
-482.161220 138.171426 translate
/vertices [
  [-0.3101021 1.3797959] % 0
  [1.3797959 0.3101021] % 1
  [-1.3797959 -0.3101021] % 2
  [0.3101021 -1.3797959] % 3
  [0.4155457 0.9204658] % 4
  [-0.6979325 -0.7417169] % 5
  [0.7318457 -0.7135271] % 6
  [0.7697969 0.6962272] % 7
  [-0.6032723 -0.8016362] % 8
  [0.6172062 -0.894634] % 9
  [0.948181 -0.3717613] % 10
  [-0.7694321 0.6541482] % 11
  [-0.1750598 -1.0726918] % 12
  [0.8882618 -0.4664215] % 13
  [-0.9936707 0.2998969] % 14
  [-0.3561668 -0.9580523] % 15
  [0.9763709 -0.3272271] % 16
  [-0.6647854 0.751411] % 17
  [-0.6647854 0.751411] % 18
  [-1 -0.4258093] % 19
  [-1 -0.4258093] % 20
  [0.7071436 -0.7525513] % 21
  [0.0696938 -1.0348469] % 22
  [0.6541482 0.7694321] % 23
  [-0.8292856 -0.6585713] % 24
  [0.0696938 -1.0348469] % 25
  [0.0696938 1.1393877] % 26
  [-0.8292856 -0.6585713] % 27
  [1.0313267 -0.2404082] % 28
  [-0.5505103 1] % 29
  [0.3797959 -0.9651531] % 30
  [1.0313267 -0.2404082] % 31
  [-0.9204658 0.4155457] % 32
  [0.3797959 -0.9651531] % 33
  [-0.844949 0.5348469] % 34
  [0.3101021 -1.3797959] % 35
  [0.3101021 -1.3797959] % 36
  [0.5348469 0.844949] % 37
  [0.5348469 0.844949] % 38
  [-0.844949 0.5348469] % 39
  [-0.3171426 -0.9827543] % 40
  [1.0860309 0.0430155] % 41
  [1.0860309 0.0430155] % 42
  [0.2793455 0.9635996] % 43
  [0.2793455 0.9635996] % 44
  [-0.7424667 -0.7135271] % 45
  [-1.0696938 -0.2404082] % 46
  [0.7468027 -0.6898979] % 47
  [0.3797959 0.9430952] % 48
  [-1.0696938 -0.2404082] % 49
  [0.7468027 -0.6898979] % 50
  [0.3797959 0.9430952] % 51
  [0.9519184 0.5038367] % 52
  [-0.8114823 0.5877175] % 53
  [-1.1393877 0.0696938] % 54
  [0.8076734 -0.5937347] % 55
  [0.8076734 -0.5937347] % 56
  [0.9519184 0.5038367] % 57
  [-1.0759592 0.0480816] % 58
  [-0.4759592 -0.8822245] % 59
  [-0.4759592 -0.8822245] % 60
  [1 0.5505103] % 61
  [0.4819764 0.8784157] % 62
  [-1.0759592 0.0480816] % 63
  [-0.3797959 -0.9430952] % 64
  [1.0696938 0.2404082] % 65
  [-0.7468027 0.6898979] % 66
  [-0.3797959 -0.9430952] % 67
  [1.0696938 0.2404082] % 68
  [-0.7468027 0.6898979] % 69
] def
/faces [
  [1 0 2 3][4 5][6 7][8 9][10 11][12 13][14 15][16 17][18 19][20 21][22 23]
  [24 25][26 27][28 29][30 31][32 33][34 35][36 37][38 39][40 41][42 43][44 45]
  [46 47][48 49][50 51][52 53][54 55][56 57][58 59][60 61][62 63][64 65][66 67]
  [68 69]
] def
/bbox [-1.3797959 -1.3797959 1.3797959 1.3797959] def
/scalingSize 50 def
.1 setlinewidth

/dx bbox 2 get bbox 0 get sub scalingSize mul def
/dy bbox 3 get bbox 1 get sub scalingSize mul def

gsave
  bbox 0 get scalingSize mul neg
  bbox 1 get scalingSize mul neg
  translate
  faces aload length { %repeat
    dup 0 get
    vertices exch get
    aload pop
    scalingSize mul exch
    scalingSize mul exch
    moveto
    aload length 1 sub { %repeat
      vertices exch get
      aload pop
      scalingSize mul exch
      scalingSize mul exch
      lineto
    } repeat
    pop
    closepath stroke
  } repeat


grestore
142.979590 0 translate
/vertices [
  [-1.4117143 0.0840408] % 0
  [0.0840408 1.4117143] % 1
  [1.4117143 -0.0840408] % 2
  [-0.0840408 -1.4117143] % 3
  [0.8081863 -0.6197488] % 4
  [1.1628417 0.0084898] % 5
  [1.1628417 0.0084898] % 6
  [-0.4558788 0.9324667] % 7
  [-1 0.2457187] % 8
  [0.7244011 -0.6941189] % 9
  [-0.7694321 0.6541482] % 10
  [-1 0.2457187] % 11
  [0.0939057 1.0469529] % 12
  [-1.0322441 0.0292856] % 13
  [-1.0322441 0.0292856] % 14
  [0.3453835 -1.0305454] % 15
  [0.5056841 -0.8882581] % 16
  [0.0939057 1.0469529] % 17
  [0.2639563 1.0038161] % 18
  [-1.1418817 -0.2199522] % 19
  [1.0028784 -0.1973673] % 20
  [0.2639563 1.0038161] % 21
  [-0.962912 -0.4215791] % 22
  [1.0028784 -0.1973673] % 23
  [-1.075551 0.3824286] % 24
  [0.924449 -0.5165509] % 25
  [0.924449 -0.5165509] % 26
  [0.930602 0.4579796] % 27
  [0.583612 0.8488981] % 28
  [-0.5582413 0.8416069] % 29
  [-0.9849778 -0.2230568] % 30
  [-0.1642655 -1.0169797] % 31
  [0.5651531 0.8696938] % 32
  [-0.9849778 -0.2230568] % 33
  [-0.1642655 -1.0169797] % 34
  [0.9121431 0.4787754] % 35
  [-0.0840408 -1.4117143] % 36
  [0.7478775 0.6638367] % 37
  [0.7478775 0.6638367] % 38
  [-0.6638367 0.7478775] % 39
  [-0.6638367 0.7478775] % 40
  [-0.0840408 -1.4117143] % 41
  [-0.9074693 0.4623345] % 42
  [-1.1418817 -0.2199522] % 43
  [-0.962912 -0.4215791] % 44
  [0.4711432 -0.9189176] % 45
  [1 -0.1411934] % 46
  [-0.9074693 0.4623345] % 47
  [0.8476042 -0.5847604] % 48
  [1 -0.1411934] % 49
  [-0.7478775 -0.6638367] % 50
  [1.2898979 0.0531973] % 51
  [1.0462653 0.3276734] % 52
  [-0.8010748 0.6260612] % 53
  [-0.8010748 0.6260612] % 54
  [-0.7478775 -0.6638367] % 55
  [1.00193 -0.0071886] % 56
  [-0.2084898 1.0169797] % 57
  [-0.4823429 -0.9629877] % 58
  [1.00193 -0.0071886] % 59
  [-0.2084898 1.0169797] % 60
  [-0.5628162 -0.8723266] % 61
  [0.6117143 -0.7941429] % 62
  [-0.2521225 1.1133265] % 63
  [-0.7106333 0.7063396] % 64
  [-0.5628162 -0.8723266] % 65
  [-0.4823429 -0.9629877] % 66
  [0.6117143 -0.7941429] % 67
  [0.2058571 1.2744762] % 68
  [-0.7478775 -0.6638367] % 69
  [-0.7478775 -0.6638367] % 70
  [0.5265986 -0.8696938] % 71
  [0.5265986 -0.8696938] % 72
  [0.4494897 1] % 73
] def
/faces [
  [3 2 1 0][4 5][6 7][8 9][10 11][12 13][14 15][16 17][18 19][20 21][22 23]
  [24 25][26 27][28 29][30 31][32 33][34 35][36 37][38 39][40 41][42 43][44 45]
  [46 47][48 49][50 51][52 53][54 55][56 57][58 59][60 61][62 63][64 65][66 67]
  [68 69][70 71][72 73]
] def
/bbox [-1.4117143 -1.4117143 1.4117143 1.4117143] def
/scalingSize 50 def
.1 setlinewidth

/dx bbox 2 get bbox 0 get sub scalingSize mul def
/dy bbox 3 get bbox 1 get sub scalingSize mul def

gsave
  bbox 0 get scalingSize mul neg
  bbox 1 get scalingSize mul neg
  translate
  faces aload length { %repeat
    dup 0 get
    vertices exch get
    aload pop
    scalingSize mul exch
    scalingSize mul exch
    moveto
    aload length 1 sub { %repeat
      vertices exch get
      aload pop
      scalingSize mul exch
      scalingSize mul exch
      lineto
    } repeat
    pop
    closepath stroke
  } repeat


grestore
146.171426 0 translate

showpage
%%EndPage

%%Tailer
%%EOF
