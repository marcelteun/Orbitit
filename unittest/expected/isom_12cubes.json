{"class": "symmetric_shape", "data": {"cols": [[0.996094, 0.839844, 0], [0.132812, 0.542969, 0.132812], [0.542969, 0, 0], [0, 0.746094, 0.996094], [0.542969, 0, 0], [0.132812, 0.542969, 0.132812], [0, 0.746094, 0.996094], [0, 0.746094, 0.996094], [0.996094, 0.839844, 0], [0.542969, 0, 0], [0.132812, 0.542969, 0.132812], [0.996094, 0.839844, 0]], "fs": [[0, 1, 2, 3], [0, 3, 7, 4], [1, 0, 4, 5], [2, 1, 5, 6], [3, 2, 6, 7], [7, 6, 5, 4]], "isometries": [{"class": "transform_3d", "data": [{"class": "quaternion", "data": [-0.5, 0.5, -0.5, -0.5]}, {"class": "quaternion", "data": [-0.5, -0.5, 0.5, 0.5]}]}, {"class": "transform_3d", "data": [{"class": "quaternion", "data": [-0.707106781187, 0, -0.707106781187, 0]}, {"class": "quaternion", "data": [-0.707106781187, 0, 0.707106781187, 0]}]}, {"class": "transform_3d", "data": [{"class": "quaternion", "data": [0, 0, -0.707106781187, -0.707106781187]}, {"class": "quaternion", "data": [0, 0, 0.707106781187, 0.707106781187]}]}, {"class": "transform_3d", "data": [{"class": "quaternion", "data": [0.707106781187, -0.707106781187, 0, 0]}, {"class": "quaternion", "data": [0.707106781187, 0.707106781187, 0, 0]}]}, {"class": "transform_3d", "data": [{"class": "quaternion", "data": [0, -0.707106781187, 0.707106781187, 0]}, {"class": "quaternion", "data": [0, 0.707106781187, -0.707106781187, 0]}]}, {"class": "transform_3d", "data": [{"class": "quaternion", "data": [0.5, -0.5, -0.5, -0.5]}, {"class": "quaternion", "data": [0.5, 0.5, 0.5, 0.5]}]}, {"class": "transform_3d", "data": [{"class": "quaternion", "data": [-0.5, -0.5, 0.5, -0.5]}, {"class": "quaternion", "data": [-0.5, 0.5, -0.5, 0.5]}]}, {"class": "transform_3d", "data": [{"class": "quaternion", "data": [0, 0.707106781187, 0, -0.707106781187]}, {"class": "quaternion", "data": [0, -0.707106781187, 0, 0.707106781187]}]}, {"class": "transform_3d", "data": [{"class": "quaternion", "data": [-0.707106781187, -0.707106781187, 0, 0]}, {"class": "quaternion", "data": [-0.707106781187, 0.707106781187, 0, 0]}]}, {"class": "transform_3d", "data": [{"class": "quaternion", "data": [0, 0.707106781187, 0, 0.707106781187]}, {"class": "quaternion", "data": [0, -0.707106781187, 0, -0.707106781187]}]}, {"class": "transform_3d", "data": [{"class": "quaternion", "data": [0, 0, -0.707106781187, 0.707106781187]}, {"class": "quaternion", "data": [0, 0, 0.707106781187, -0.707106781187]}]}, {"class": "transform_3d", "data": [{"class": "quaternion", "data": [0.707106781187, 0, -0.707106781187, 0]}, {"class": "quaternion", "data": [0.707106781187, 0, 0.707106781187, 0]}]}], "name": "isom_12cubes", "orientation": {"class": "transform_3d", "data": [{"class": "quaternion", "data": [0.953020613871, 0.214186495298, 0, 0.214186495298]}, {"class": "quaternion", "data": [0.953020613871, -0.214186495298, 0, -0.214186495298]}]}, "vs": [[1, 1, 1], [-1, 1, 1], [-1, -1, 1], [1, -1, 1], [1, 1, -1], [-1, 1, -1], [-1, -1, -1], [1, -1, -1]]}}